package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "dist": "rm -rf dist; node esbuild.js",
+    "lint": "eslint --fix src/",
```

Comment:
Ahhh I think there should be no `--fix` in `lint`. Maybe we can put --fix in a separate script like `lint-fix`